Yoast/YoastSEO.js
# Add Polish passive voice

## Summary

This PR can be summarized in the following changelog entry:

* We should add passive voice assessment for Polish


## Test instructions

This PR can be tested by following these steps:

* Build a beta script using trunk for Yoast SEO and this branch for YoastSEO.js.
* Copy and paste a Polish text from e.g. Wikipedia. See if passive voice is detected (if there are no passive sentences, this might also be due to the text; try a different text in that case).
* Add the following sentences, all of which should be marked as passive:

Jabłko zostało zjedzone.
Jabłko jest zjedzone.
Jabłko zjedzone było.
Jabłko może być zjedzone.

* Add the following sentences, none of which should be marked as passive:

Ktoś został królem.
Jestem wysoka.
Gdy komuś jest gorąco może lubić otwarte okna. 


Fixes #1614
